web/src/components/snapshots/SnapshotSchedule.jsx

```diff
@@ -449,33 +455,35 @@ class SnapshotSchedule extends Component {
             </div>
           )}
           <div className="flex flex-column snapshot-form-wrapper card-bg u-padding--15">
-            <p className="card-title">Automatic snapshots</p>
+            <p className="card-title">Automatic {featureName}s</p>
             <div className="u-marginBottom--10">
               <p className="u-fontSize--normal u-fontWeight--normal u-lineHeight--normal u-textColor--bodyCopy u-marginTop--12 schedule">
-                Set up a custom schedule and retention policy for automatic
-                snapshots of the Admin Console and all application data.
+                Configure a schedule and retention policy for automatic{" "}
+                {featureName}s of the admin console and all application data.
```

Comment:
I might remove this because the retention policy doesn't apply to just automatic backups.

I'd still support making retention policy it's own box because of that, but it's nbd for now.
```suggestion
                Configure a schedule and retention policy for {" "}
                {featureName}s of the admin console and all application data.
```